search/django/weedid/urls.py

```diff
@@ -2,6 +2,16 @@
 from weedid import views
 
 urlpatterns = [
-    path("api/", views.test, name="test"),
+    path("api/test/", views.test, name="test"),
+    path("api/upload/", views.upload, name="upload"),
+    path("api/upload_image/", views.upload_image, name="upload_image"),
+    path("api/submit_deposit/", views.submit_deposit, name="submit_deposit"),
+    path("api/upload_status/", views.upload_status, name="upload_status"),
+    path("api/upload_info/", views.upload_info, name="upload_info"),
+    path("api/upload_list/", views.upload_list, name="upload_list"),
```

Comment:
I wonder if we should make these paths look more RESTful. For example:
* upload: `POST upload`
* upload_image: `PUT upload/<ID>/image`
* submit_deposit: `POST upload/<ID>/complete`
* upload_status: `GET upload`
* upload_info: `POST upload/<ID>/meta`
* upload_list: `GET uploads`

WDYT?? I'm no expert in REST design.

Reply:
Coz we don't actually expose these entities as api and only talk to them with our frontend. Including the id and other stuffs in the payload is easier to keep consistency of frontend development. Shall we postpone this till having it fully functional?

Reply:
Yes, not public facing, so we'll not prioritise this. But it will be confusing for maintainers.